### ISSUE TITLE
Add integration test coverage to the upgrader class

### ISSUE THREAD

**Issue by danielbachhuber** [easydigitaldownloads/EDD-License-handler#23]
Given how critical a piece of software the upgrader class is, it would be ideal to have integration test coverage for it.

**Comment by danielbachhuber** [comment]
On this note, it would be helpful to have tests we could add to our own plugin to verify all of the integration aspects were working as expected.

**Comment by pippinsplugins** [comment]
I agree 100%. Hopefully we'll be writing a much better sample integration in the near future that could include full unit tests.
